h2o-core/src/main/java/water/util/Countdown.java

```diff
@@ -32,11 +32,11 @@ public Countdown(long _time_limit_millis, boolean start) {
     this(_time_limit_millis);
     if (start) start();
   }
-  
+
   public Date start_time() {
     return new Date(_start_time);
   }
-  
+
```

Comment:
@seb-h2o it is not your changes but shouldn't we use here camel case for methods? Inconsistency within class... for example `fromSeconds` vs `start_time`

Reply:
right right right, will do a small countdown renaming in a small future PR